engines/registry/app/models/access_scope.rb

```diff
@@ -91,8 +91,11 @@ def allowed_paths(system = nil)
     if system && system.hybrid?
       allowed_product_class = allowed_products.detect { |s| s.downcase.include?('ltss') }
```

Comment:
It looks weird to hardcode 'ltss' here. Why is this not compared with the actual product class?

Reply:
I want to check if the product needs to check subscription with SCC
the product class is, i.e., `SLES15-SP4-LTSS-X86`  if Im not mistaken
so, checking for `ltss` to include any version or arch made sense to me

Reply:
using a check for non free product